client/constants/messages.js

```diff
@@ -103,7 +103,27 @@ const messages = {
    * webview -> browser
    * webview sends this to delete all site settings.
    */
-  DELETE_SYNC_SITE_SETTINGS: _
+  DELETE_SYNC_SITE_SETTINGS: _,
+  /**
+   * browser -> webview
+   * browser sends this to get base bookmarks order for the particular device.
```

Comment:
does this also need to send the device type? (Mobile vs Desktop)
@davidtemkin's spec says "Desktop devices come first, followed by mobile devices."

Reply:
NVM, i see it's the platform string.